include/wx/webview.h

```diff
@@ -223,6 +223,10 @@ class WXDLLIMPEXP_WEBVIEW wxWebView : public wxControl
 protected:
     virtual void DoSetPage(const wxString& html, const wxString& baseUrl) = 0;
 
+    //Counts number of calls of RunScript, to avoid
```

Comment:
It was a @vadz idea. AFAIK, the main purpose of it is if you call more than one RunScript, you could lose some information because of variable overlap. IDK how this could happen because it runs synchronously, but it is not a bad idea and it is not a big deal on the code.